pkg/cmd/admin/router/router.go

```diff
@@ -767,5 +768,6 @@ func validateServiceAccount(client *kclient.Client, ns string, serviceAccount st
 		}
 	}
 
-	return fmt.Errorf("service account %q is not allowed to access the host network on nodes, needs access via a security context constraint", serviceAccount)
+	errMsg := "service account %q is not allowed to access the host network on nodes, grant access with oadm policy add-scc-to-user %s system:serviceaccount:%s:%s"
```

Comment:
Also remember you might have a newer admin command than server, so the message may be surprising